src/apps/openfluid-devstudio/MainWindow.cpp

```diff
@@ -456,87 +526,114 @@ void MainWindow::createMenus()
 // =====================================================================
 
 
-void MainWindow::setWorkspaceDefaults()
+void MainWindow::updateBuildJobsStatus()
 {
-  auto* WMgr = openfluid::base::WorkspaceManager::instance();
+  QString JobStatusString;
+  if (m_WareBuildOptions.m_IsParallelJobs)
+  {
+    JobStatusString = QString::fromStdString(std::to_string(m_WareBuildOptions.m_JobsNumber))+" "+tr("jobs");
```

Comment:
text to use: "parallel build using x jobs"

use QString.arg() for number conversion instead of using std::string